week1/homework/exercises/3-web-server/server.js

```diff
@@ -1,14 +1,34 @@
 /**
  * Exercise 3: Create an HTTP web server
  */
+const http = require('http');
+const { readFileSync } = require(`fs`);
 
-var http = require('http');
-
-//create a server
-let server = http.createServer(function (req, res) {
-  // YOUR CODE GOES IN HERE
-	res.write('Hello World!'); // Sends a response back to the client
-	res.end(); // Ends the response
+const server = http.createServer((req, res) => {
+  if (req.url === '/') {
+    const localHost = readFileSync('./index.html');
+    res.writeHead(200, { 'content-type': 'text/html' });
+    res.write(localHost);
+    res.end();
+  } else if (req.url === '/index.js') {
+    const logic = readFileSync('./index.js');
+    res.writeHead(200, { 'content-type': 'application/javascript' });
+    res.write(logic);
+    res.end();
+  } else if (req.url === '/style.css') {
+    const cssStyle = readFileSync('./style.css');
+    res.writeHead(200, { 'content-type': 'text/css' });
+    res.write(cssStyle);
+    res.end();
+  } else if (req.url === '*') {
```

Comment:
```suggestion
  } else {
```
No need to specify a condition, we can just catch all the rest 👍 

Great work with the `404` though!